src/logging.c

```diff
@@ -210,6 +211,44 @@ void log_connect_to_buffer(bool connected, PgSocket *client) {
   len += sizeof(uint8_t);
 }
 
+
+/*
+ * Log skipped packet to buffer.
+ */
+void log_pkt_skipped_to_buffer(PktHdr *pkt, PgSocket *client) {
```

Comment:
I think we can log the query as well, so we know what a long query looks like

Reply:
I agree that we should be adding the queries to this but we currently don't have any way to use these in a useful way in the replayer.

I can create a separate PR for replayer and bouncer that uses these queries but for now the current changes get us a lot of value.